core/2d/SpriteFrameCache.cpp

```diff
@@ -170,6 +170,33 @@ void SpriteFrameCache::removeUnusedSpriteFrames()
     }
 }
 
+void SpriteFrameCache::removeUnusedSpriteSheetFile()
```

Comment:
This method name, `removeUnusedSpriteSheetFile`, is a little misleading, and perhaps it's better to call it `remoteUnusedSpriteSheets`, since that is exactly what it does.

Reply:
modified